src/Microsoft.AspNet.Identity/IdentityOptions.cs

```diff
@@ -2,8 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNet.Identity;
 
-namespace Microsoft.AspNet.Identity
+namespace Microsoft.AspNet.Builder
```

Comment:
Yes since it's inconvenient otherwise when users have `app.UseMiddleware(new MiddlewareOptions());`. Moving options to the same namespace as `UseMiddleware` removes the need to add another using statement.